### ISSUE TITLE
correct column values?

### ISSUE THREAD

**Issue by claytonpbarrows** [NREL-SIIP/PyPSA2PowerSystems.jl#3]
particularly for generator capacities, max load values, and max flow limits?

**Comment by claytonpbarrows** [comment]
The usage of columns is defined here: https://github.com/NREL-SIIP/PyPSA2PowerSystems.jl/blob/main/deps/user_descriptors.yaml

**Comment by pz-max** [comment]
The documentation on each component: https://pypsa.readthedocs.io/en/latest/components.html#generator

- Netcdf files of the **unsolved** and **solved** network https://drive.google.com/drive/folders/1Bou4vXg3lovfLWyqVYqnbHTMuTCdM9qJ?usp=sharing

In case your import function has problems, you can import the network file by:
``` bash
import pypsa

filename = "elec_s_1024_ec.nc" # example
n = pypsa.Network(filename)
```

As explained in the documentation, you can access every listed variable easily. Here some examples:
```bash
n.generators.p_nom_opt (optimized active power size of the plant - static value)
n.generators_t.p (optimized active power time series - _t signal time series)
n.loads.p_set 
n.loads_t.p  
```

**Comment by claytonpbarrows** [comment]
This is mostly resolved in 7e6be907029af286e6bfa4173e944f159ccf5b6c
